### DISCUSSION TITLE
deformation field

### DISCUSSION THREAD

**Discussion by Marie0704** [Project-MONAI/tutorials#1515] [Q&A]
Is it possible to save the model output as a deformation field? If so, please advise how to do it.

**Comment by KumoLiu** [comment]
Hi, perhaps you can refer to:
https://github.com/Project-MONAI/tutorials/blob/main/3d_registration/paired_lung_ct.ipynb
https://github.com/Project-MONAI/tutorials/blob/main/deep_atlas/deep_atlas_tutorial.ipynb
Thanks!
